### ISSUE TITLE
Can't change nickname (re-login)

### ISSUE THREAD

**Issue by jamerlan** [springlobby/springlobby#233]
on the latest development version (commit a0612759dd30ef302229e683f2f5e9e3807063f6)

I can't change nickname. I have 2 accounts: [teh]decay and [DoR]Saruman for example.
I press "Disconnect" and then "Connect". My current account is [teh]decay. When I open "Connect" dialogue I want to type an other nickname([DoR]Saruman) and I can't do that. Dialogie window closes automatically and I logged in as [teh]decay again. 

In short: Dialogue window "Connect" should not auto close and auto connect.
